modules/ROOT/pages/ai-azure.adoc

```diff
@@ -0,0 +1,215 @@
+= Azure OpenAI Service integration guide
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+:navtitle: Azure OpenAI Service integration guide
+:description_short: {pluginname} with Azure OpenAI Service
+:description: A guide for integrating {pluginname} plugin using Azure OpenAI Service.
+:keywords: example, demo, custom, plugin, ai, assistant, guide, azure, openai, service
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+== Introduction
+
+This guide provides instructions for integrating the {pluginname} plugin using the https://learn.microsoft.com/en-us/azure/ai-services/openai/[Azure OpenAI Service] in {productname}.
+
+Azure OpenAI Service provides REST API access to OpenAI's language models with Azure specific features such as private networking, regional availability, and responsible AI content filtering.
+
+Azure OpenAI co-develops the APIs with OpenAI. Therefore, the following examples should be compatible if using Azure OpenAI REST API endpoints or OpenAI API endpoints.
+
+To learn more about the difference between string and streaming responses, see xref:ai.adoc#the-respondwith-object[The `respondWith` object] on the plugin page.
+
+== Prerequisites
+
+Before beginning, make sure an OpenAI model has been set up in Azure. The model's endpoint URL and API key are required for integration. For more information, see the https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart?tabs=command-line%2Cpython-new&pivots=rest-api[Quick Start Guide] provided by Azure.
+
+A proxy is required if making `fetch` calls directly to the Azure OpenAI REST API as otherwise CORS errors arise. In general, using a proxy is highly recommended even if network errors did not occur. See the xref:ai-proxy.adoc[AI Proxy Server reference guide] for information on how to setup a proxy server for use with the {pluginname}.
+
+NOTE: The xref:ai-proxy.adoc[AI Proxy Server reference guide] is, as its name notes, a reference. There is no single proxy server setup that is right or correct for all circumstances and other setups may be better for your use-case.
+
+== String response
+
+This example demonstrates how to integrate the {pluginname} plugin with the https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions[Azure OpenAI Service Chat completions API] to generate a string response.
+
```

Comment:
```suggestion

include::partial$misc/admon-ai-proxy.adoc[]

```

Reply:
This should go above the `== String response` heading

Reply:
yes, i will open a new suggestion.